asr.py

```diff
@@ -11,50 +25,111 @@
 
 logger = logging.getLogger(__name__)
 os.environ["HF_HOME"] = model_base_dir  # change dir where model is downloaded
+my_version = pkg_resources.get_distribution(
```

Comment:
This is really interesting. How does this work to get the worker version? We have struggled previously with trying to put in a Github version but we had trouble getting that into the Dockerfile.

Reply:
This approach doesn't put in the Github version. It simply obtains the version from the `pyproject.toml` file. And also, this approach wasn't actually working, so I'm replacing it with a different one that actually reads `pyproject.toml` and outputs what is written in the version field.